File: src/test/java/com/dongtronic/diabot/util/DateTest.java
package com.dongtronic.diabot.util;

import org.junit.Assert;
import org.junit.Test;

import java.sql.Timestamp;
import java.util.Date;

public class DateTest {

  public DateTest(){}

  @Test
  public void stringToDate() {
    long timestamp = 1539672587884L;
    Timestamp date = new Timestamp(timestamp);

    Assert.assertNotNull(date);
  }
}

File: bot/src/test/java/com/dongtronic/diabot/util/DateTest.kt
package com.dongtronic.diabot.util

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import java.sql.Timestamp

class DateTest {

    @Test
    fun stringToDate() {
        val timestamp = 1_539_672_587_884L
        val date = Timestamp(timestamp)

        Assertions.assertNotNull(date)
    }
}
